eng/targets/CSharp.Common.props

```diff
@@ -14,14 +14,23 @@
     <PackageReference Condition="'$(DotNetBuildFromSource)' != 'true'" Include="Microsoft.DotNet.GenAPI" PrivateAssets="All" Version="$(MicrosoftDotNetGenApiPackageVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' OR '$(RazorSdkCurrentVersionProps)' != ''">
+  <ItemGroup Condition="'$(UseRazorSdk)' == 'true' OR '$(RazorSdkCurrentVersionProps)' != ''">
```

Comment:
Maybe we can parse the whether the razor sdk is without needing to explicitly set this property. Maybe we can do something with `<Project Sdk="Microsoft.NET.Sdk.Razor">`.

Reply:
I'm actually why the original condition is specified the way it is. I don't think all of our projects using Microsoft.NET.Sdk.Web need to import the razor SDK props. In fact I think only the projects using Microsoft.NET.Sdk.Razor should need the extra imports. If there isn't a compelling reason to have almost all of our projects import the razor sdk props, I'd prefer to only import for the projects that actually use the SDK.

Reply:
WebSDK imports the Razor SDK. We need this RazorSDK from this repo to be used if you're referencing the WebSDK